internal/component/statefulset/statefulset.go

```diff
@@ -309,10 +309,17 @@ func shouldRequeueForMultiNodeEtcdIfPodsNotReady(sts *appsv1.StatefulSet) bool {
 		sts.Status.ReadyReplicas < *sts.Spec.Replicas
 }
 
-func isStatefulSetTLSConfigInSync(etcd *druidv1alpha1.Etcd, sts *appsv1.StatefulSet) bool {
+func (r _resource) hasTLSEnablementForPeerURLReflectedOnSTS(etcd *druidv1alpha1.Etcd, existingSts *appsv1.StatefulSet) bool {
+	newEtcdWrapperPeerTLSVolMounts := getEtcdContainerPeerVolumeMounts(etcd)
+	containerPeerTLSVolMounts := utils.GetEtcdContainerPeerTLSVolumeMounts(existingSts)
```

Comment:
```suggestion
	containerPeerURLTLSVolMounts := utils.GetEtcdContainerPeerURLTLSVolumeMounts(existingSts)
```
do you want to change the name as we discussed the PeerTLS is technically wrong name to say.

Reply:
`containerPeerTLSVolMounts` is semantically correct.